src/core_plugins/kibana/public/dashboard/panel/panel.html

```diff
@@ -1,4 +1,11 @@
-<div class="panel panel-default" ng-switch on="panel.type" ng-if="savedObj || error">
+<div
+  class="panel panel-default"
+  ng-switch on="panel.type"
+  ng-if="savedObj || error"
+  reporting-item
+  reporting-item-title="{{savedObj.title}}"
+  reporting-item-description="{{savedObj.description}}"
```

Comment:
I'm afraid that just `item-` might be too generic, I feel like `vis-` is closer, but it would be added to saved searches/discover which in my mind weren't just visualizations.